packages/vaex-ml/setup.py

```diff
@@ -12,7 +12,18 @@
 license     = 'MIT'
 version     = version.__version__
 url         = 'https://www.github.com/vaexio/vaex'
-install_requires_ml = ['vaex-core>=4.8.0,<5', 'numba', 'traitlets', 'jinja2']
+install_requires_ml = [
+      'vaex-core~=4.8',
+      'numba',
+      'traitlets',
+      'jinja2',
+      'annoy',
+      'scikit-learn',
+      'xgboost',
+      'lightgbm~=4.0',
+      'catboost',
```

Comment:
these should be optional right? put them under all?

Reply:
which ones exactly should go to `all`?

Reply:
all these, since they were not a dependency before (more like a peer dependency - in npm speak)